test/inductor/test_fp8.py

```diff
@@ -0,0 +1,50 @@
+# Owner(s): ["module: inductor"]
```

Comment:
Kinda copy pasta here, if you think this test belongs in a better location let me know. I think that this may become a useful test file as we build out fp8 support